lib/t/matches.ex

```diff
@@ -66,14 +66,26 @@ defmodule T.Matches do
     Multi.update_all(multi, :bump_likes_count, query, inc: [times_liked: 1])
   end
 
-  defp maybe_notify_match(%Match{id: match_id}, by_user_id, user_id) do
-    broadcast_from_for_user(by_user_id, {__MODULE__, :matched, %{id: match_id, mate: user_id}})
-    broadcast_for_user(user_id, {__MODULE__, :matched, %{id: match_id, mate: by_user_id}})
+  defp maybe_notify_match(
+         %Match{id: match_id, expiration_date: expiration_date},
+         by_user_id,
+         user_id
+       ) do
+    broadcast_from_for_user(
+      by_user_id,
+      {__MODULE__, :matched, %{id: match_id, mate: user_id, expiration_date: expiration_date}}
```

Comment:
adding expiration_date to every push / reply for client to always have the updated expiration_date